libr/asm/arch/pyc/opcode.c

```diff
@@ -0,0 +1,289 @@
+	{ "2.3a0", opcode_23 },
+	{ "2.4a0", opcode_24 },
+	{ "2.4a2", opcode_24 },
+	{ "2.4a3", opcode_24 },
+	{ "2.5a0", opcode_25 },
+	{ "2.5b2", opcode_25 },
+	{ "2.5c3", opcode_25 },
+	{ "2.6a0", opcode_26 },
+	{ "2.6a1+", opcode_26 },
+	{ "2.7a0", opcode_27 },
+	{ "2.7a2+", opcode_27 },
+	{ "3.0a1", opcode_30 },
+	{ "3.0a1+", opcode_30 },
+	{ "3.0a2", opcode_30 },
+	{ "3.0a2+", opcode_30 },
+	{ "3.0a3+", opcode_30 },
+	{ "3.0a5+", opcode_30 },
+	{ "3.0x", opcode_30 },
+	{ "3.1a0", opcode_31 },
+	{ "3.2a0", opcode_32 },
+	{ "3.2a1+", opcode_32 },
+	{ "3.2a2+", opcode_33 },
+	{ "3.3.0a0", opcode_33 },
+	{ "3.3.0a1+", opcode_33 },
+	{ "3.3.0a3+", opcode_33 },
+	{ "3.3a0", opcode_33 },
+	{ "3.4.0a0", opcode_34 },
+	{ "3.4.0a3+", opcode_34 },
+	{ "3.4.0rc1+", opcode_34 },
+	{ "3.5.0a0", opcode_35 },
+	{ "3.5.0a4+", opcode_35 },
+	{ "3.5.0b1+", opcode_35 },
+	{ "3.5.0b2+", opcode_35 },
+	{ "3.6.0a0", opcode_36 },
+	{ "v3.6.0", opcode_36 },
+	{ "v3.6.0a2", opcode_36 },
+	{ "v3.6.0a3", opcode_36 },
+	{ "v3.6.0a4", opcode_36 },
+	{ "v3.6.0b1", opcode_36 },
+	{ "v3.6.0b2", opcode_36 },
+	{ "v3.6.0b3", opcode_36 },
+	{ "v3.6.0b4", opcode_36 },
+	{ "v3.6.0rc1", opcode_36 },
+	{ "v3.6.0rc2", opcode_36 },
+	{ "v3.6.1", opcode_36 },
+	{ "v3.6.10", opcode_36 },
+	{ "v3.6.10rc", opcode_36 },
+	{ "v3.6.1rc1", opcode_36 },
+	{ "v3.6.2", opcode_36 },
+	{ "v3.6.2rc1", opcode_36 },
+	{ "v3.6.2rc2", opcode_36 },
+	{ "v3.6.3", opcode_36 },
+	{ "v3.6.3rc1", opcode_36 },
+	{ "v3.6.4", opcode_36 },
+	{ "v3.6.4rc1", opcode_36 },
+	{ "v3.6.5", opcode_36 },
+	{ "v3.6.5rc1", opcode_36 },
+	{ "v3.6.6", opcode_36 },
+	{ "v3.6.6rc1", opcode_36 },
+	{ "v3.6.7", opcode_36 },
+	{ "v3.6.7rc1", opcode_36 },
+	{ "v3.6.7rc2", opcode_36 },
+	{ "v3.6.8", opcode_36 },
+	{ "v3.6.8rc1", opcode_36 },
+	{ "v3.6.9", opcode_36 },
+	{ "v3.6.9rc1", opcode_36 },
+	{ "v3.7.0", opcode_37 },
+	{ "v3.7.0a1", opcode_37 },
+	{ "v3.7.0a2", opcode_37 },
+	{ "v3.7.0a3", opcode_37 },
+	{ "v3.7.0a4", opcode_37 },
+	{ "v3.7.0b1", opcode_37 },
+	{ "v3.7.0b2", opcode_37 },
+	{ "v3.7.0b3", opcode_37 },
+	{ "v3.7.0b4", opcode_37 },
+	{ "v3.7.0b5", opcode_37 },
+	{ "v3.7.0rc1", opcode_37 },
+	{ "v3.7.1", opcode_37 },
+	{ "v3.7.1rc1", opcode_37 },
+	{ "v3.7.1rc2", opcode_37 },
+	{ "v3.7.2", opcode_37 },
+	{ "v3.7.2rc1", opcode_37 },
+	{ "v3.7.3", opcode_37 },
+	{ "v3.7.3rc1", opcode_37 },
+	{ "v3.7.4", opcode_37 },
+	{ "v3.7.4rc1", opcode_37 },
+	{ "v3.7.4rc2", opcode_37 },
+	{ "v3.7.5", opcode_37 },
+	{ "v3.7.5rc1", opcode_37 },
+	{ "v3.7.6", opcode_37 },
+	{ "v3.7.6rc1", opcode_37 },
+	{ "v3.8.0", opcode_38 },
+	{ "v3.8.0a1", opcode_38 },
+	{ "v3.8.0a2", opcode_38 },
+	{ "v3.8.0a3", opcode_38 },
+	{ "v3.8.0a4", opcode_38 },
+	{ "v3.8.0b1", opcode_38 },
+	{ "v3.8.0b2", opcode_38 },
+	{ "v3.8.0b3", opcode_38 },
+	{ "v3.8.0b4", opcode_38 },
+	{ "v3.8.0rc1", opcode_38 },
+	{ "v3.8.1", opcode_38 },
+	{ "v3.8.1rc1", opcode_38 },
+	{ "v3.9.0a1", opcode_39 },
+	{ "v3.9.0a2", opcode_39 },
+	{ "v3.9.0a3", opcode_39 },
```

Comment:
i would suggesting adding a {NULL, NULL} entry at the end

Reply:
Good!